napalm_ios/ios.py

```diff
@@ -30,6 +30,14 @@
 WEEK_SECONDS = 7 * DAY_SECONDS
 YEAR_SECONDS = 365 * DAY_SECONDS
 
+# IP Address
+IP_ADDR_REGEX = r"\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}"
+RE_IPADDR_STRIP = re.compile(r"({})\n".format(IP_ADDR_REGEX))
+
+IOS_COMMANDS = {
+   'show_mac_address': ['show mac-address table', 'show mac address-table'],
```

Comment:
The first command is actually wrong here and it wouldn't get picked up by the tests. It should be `show mac-address-table`.